addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/settings/YamahaZoneSettings.java

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
More of a note: The framework and other bindings prefer "configuration" over "settings" or "preferences". What do you thing?

Reply:
Ok, will change to "Configuration" for sake of consistency.